activiti-core/activiti-engine/src/main/java/org/activiti/engine/impl/persistence/entity/TaskEntityManager.java

```diff
@@ -59,4 +61,7 @@ public interface TaskEntityManager extends EntityManager<TaskEntity> {
 
   void deleteTask(TaskEntity task, String deleteReason, boolean cascade, boolean cancel);
 
+  TaskInfo getHistoricTask(TaskEntity task);
```

Comment:
TaskEntityManager should be responsible for retrieving historical tasks.

Reply:
Sorry but I don't understand: it's the `TaskEntityManager` that's responsible for retrieving historical tasks.

Reply:
I meant shouldn't :)

Reply:
Ah ok! Maybe on a higher abstraction level.. Or I just do what you said, don't refactor that code and use directly `HistoryManager`, without the getHistoricalTask. Do you agree?